openedx/core/djangoapps/bookmarks/tests/test_models.py

```diff
@@ -48,9 +48,9 @@ def setUp(self):
         self.admin = AdminFactory()
         self.user = UserFactory.create(password=self.TEST_PASSWORD)
         self.other_user = UserFactory.create(password=self.TEST_PASSWORD)
-        self.setup_test_data(self.STORE_TYPE)
+        self.setup_data(self.STORE_TYPE)
 
-    def setup_test_data(self, store_type=ModuleStoreEnum.Type.mongo):
+    def setup_data(self, store_type=ModuleStoreEnum.Type.mongo):
```

Comment:
Note to reviewers: this helper method was inadvertently being executed as a test point.